core/tutorials/Unloading/Plugin/PluginClass.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Runtime.Loader;
+
+namespace Plugin
+{
+    public class PluginClass : Interface
+    {
+        static GCHandle pluginHandle;
+
+        public static Interface GetInterface()
+        {
+            PluginClass plugin = new PluginClass();
+
+            // We create the GC handle just to demonstrate how we can do necessary cleanup before
+            // unloading. Creating a normal GC handle for the plugin instance would prevent unloading
+            // from succeeding.
+            pluginHandle = GCHandle.Alloc(plugin);
```

Comment:
What most people do with the same is to copy&paste into their project. I do not think we want to create the GCHandle for "demonstration purposes".

Reply:
Perhaps the callback could simply write to stdout - `"Clean up here"`?